src/contracts/core/DelegationManager.sol

```diff
@@ -50,7 +55,7 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         initializer
     {
         _initializePauser(_pauserRegistry, initialPausedStatus);
-        DOMAIN_SEPARATOR = keccak256(abi.encode(DOMAIN_TYPEHASH, bytes("EigenLayer"), block.chainid, address(this)));
+        DOMAIN_SEPARATOR = keccak256(abi.encode(DOMAIN_TYPEHASH, bytes("EigenLayer"), ORIGINAL_CHAIN_ID, address(this)));
```

Comment:
think we should also account for 4.5.B here -- using `keccak256(bytes("EigenLayer"))` instead of `bytes("EigenLayer")` to better match EIP-712.
Should also perhaps discuss 4.5.C

Reply:
addressed.